pandas/core/frame.py

```diff
@@ -5205,8 +5205,10 @@ def swaplevel(self, i=-2, j=-1, axis=0):
 
         Parameters
         ----------
-        i, j : int, str (can be mixed)
-            Level of index to be swapped. Can pass level name as string.
+        i : int, str (can be mixed)
```

Comment:
`int or str`, not sure what the can be mixed means here, but shouldn't be in the type (can be explained in the description)